Sprint-3/implement/get-card-value.js

```diff
@@ -29,3 +29,19 @@
 // Given a card with an invalid rank (neither a number nor a recognized face card),
 // When the function is called with such a card,
 // Then it should throw an error indicating "Invalid card rank."
+
+const getCardValue = (card) => {
+    const rank = card.slice(0, -1); 
+    if (!isNaN(rank) && rank >= 2 && rank <= 10) 
+        return parseInt(rank, 10); 
+    if (['J', 'Q', 'K'].includes(rank)) 
+        return 10; 
+    if (rank === 'A') 
+        return 11; 
+    throw new Error("Invalid card rank"); 
+    /*
+    const error = new Error("Invalid card rank");  //create the error
```

Comment:
Why is this commented out?

Reply:
I have removed the comment It does no thing here.That it the same with the throw new Error. I use to check in different syntax of the error